generators/server/files.js

```diff
@@ -1718,6 +1718,10 @@ const serverFiles = {
                     file: 'package/service/MailServiceIT.java',
                     renameTo: generator => `${generator.testDir}service/MailServiceIT.java`
                 },
+                {
```

Comment:
Move it to https://github.com/jhipster/generator-jhipster/blob/3d1cc2c2df5599be7ee73247f56f30cd3074fadf/generators/server/files.js#L1534, so this keep consistancy and is created only when needed.